mcs/class/System.Xml.Linq/Test/System.Xml.Schema/ExtensionsTest.cs

```diff
@@ -220,8 +220,8 @@ public void XDocumentAddSchemaInfoValidate ()
 				}
 			}
 
-			AssertHelper.GreaterOrEqual (afterNoOfAttributes, beforeNoOfAttributes, "newAttributes");
-			AssertHelper.GreaterOrEqual (afterNoOfElements, beforeNoOfElements, "newElements");       
+			Assert.IsTrue (afterNoOfAttributes >= beforeNoOfAttributes, "newAttributes");
+			Assert.IsTrue (afterNoOfElements >= beforeNoOfElements, "newElements");
```

Comment:
same here, include afterNoOfElements and beforeNoOfElements in the assert message